server/app/views/applicant/TIAccountSettingsView.java

```diff
@@ -0,0 +1,144 @@
+package views.applicant;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.each;
+import static j2html.TagCreator.h1;
+import static j2html.TagCreator.h3;
+import static j2html.TagCreator.table;
+import static j2html.TagCreator.tbody;
+import static j2html.TagCreator.td;
+import static j2html.TagCreator.th;
+import static j2html.TagCreator.thead;
+import static j2html.TagCreator.tr;
+
+import com.google.common.base.Strings;
+import com.google.inject.Inject;
+import com.typesafe.config.Config;
+import j2html.tags.specialized.DivTag;
+import j2html.tags.specialized.TdTag;
+import j2html.tags.specialized.TheadTag;
+import j2html.tags.specialized.TrTag;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+import models.AccountModel;
+import models.TrustedIntermediaryGroupModel;
+import org.slf4j.LoggerFactory;
+import play.i18n.Messages;
+import play.mvc.Http;
+import play.twirl.api.Content;
+import services.MessageKey;
+import services.applicant.ApplicantPersonalInfo;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.admin.ti.TrustedIntermediaryGroupListView;
+import views.components.ToastMessage;
+import views.style.BaseStyles;
+import views.style.ReferenceClasses;
+
+public class TIAccountSettingsView extends BaseHtmlView {
```

Comment:
**suggestion:** Seems like the other pages spell out "TrustedIntermediary" or do pascal casing with TI to be "Ti". We should probably make this consistent with one of those standards